libindy_vdr/src/state_proof/types.rs

```diff
@@ -1,31 +1,24 @@
-/**
- Single item to verification:
- - SP Trie with RootHash
- - BLS MS
- - set of key-value to verify
-*/
+/// A single result from a state proof verification
```

Comment:
I think this structure is rather input for state proof verification parsed from incoming JSON. But it is not a final result yet

Reply:
Updated